core/modules/qdisp/QdispPool.cc

```diff
@@ -100,21 +102,22 @@ void PriorityQueue::queCmd(PriorityCommand::Ptr const& cmd, int priority) {
     _cv.notify_one();
 }
 
-std::atomic<unsigned int> localLogLimiter(0); /// &&& delete
+
+std::atomic<unsigned int> localLogLimiter(0);
 
 util::Command::Ptr PriorityQueue::getCmd(bool wait){
     util::Command::Ptr ptr;
     std::unique_lock<std::mutex> uLock(_mtx);
     while (true) {
         _changed = false;
-        LOGS(_log, LOG_LVL_DEBUG, "priQueGet " << *this);
-        { /// &&& delete block
-            ++localLogLimiter;
-            if (localLogLimiter%25 == 0) {
-                LOGS(_log, LOG_LVL_WARN, "priQueGet " << *this); // &&&
-            }
+        ++localLogLimiter;
+        if (localLogLimiter%50 == 0) {
```

Comment:
What is this "magic" number 50?
Also, it would be easier to read this:
```
        if (localLogLimiter % 50 == 0) {
```
It would be also nice if this logic for separating what's reported at which log level is so important here.

Reply:
I was hoping the name would be self explanatory. It gives a good indication of the czar's work load but logging it every time is a waste of disk space, unless you're logging everything anyway. Comment added.